### ISSUE TITLE
CustomTypes - Does not export when the uid value contains a forward slash "/"

### ISSUE THREAD

**Issue by tochiedev** [boazpoolman/strapi-plugin-config-sync#56]
### Describe the bug

This happened when using customType in the plugin config. Not all data was exported to json.

### Steps to reproduce the behavior

When the value of the "uid" field value contains a forward slash e.g  "Boy / Girl", it does not export this database row to json file.

### Expected behavior

Should export "uid" field values with a forward slash.

### System

- Node.js version: ">=12.x.x <=16.x.x"
- NPM version: ">=6.0.0"
- Strapi version: "4.1.6"
- Plugin version: "^1.0.1"
- Database: Postgres
- Operating system: Mac

**Comment by boazpoolman** [comment]
Hey @tochiedev 

Thanks for reporting the issue.

On Unix-like systems, `/` is reserved. On Windows, [<>:"/\|?*](http://msdn.microsoft.com/en-us/library/aa365247%28VS.85%29#naming_conventions) along with trailing periods are reserved.

I've allready once escaped the `:` character and replaced it with `#` for windows.
I can add another one to escape `/` and replace it with `$`.

Can you test the solution I've described above in this PR #57?

```
yarn add boazpoolman/strapi-plugin-config-sync#pull/57/head
npm install boazpoolman/strapi-plugin-config-sync#pull/57/head
```

**Comment by boazpoolman** [comment]
Hi @tochiedev 

Have you had time to test out the PR I've made for this issue?

**Comment by tochiedev** [comment]
Hi @boazpoolman 

Apologies! Sorry, I couldn't get back earlier. I was working on another project and only just had the chance to test it out.

Yes, the PR solved the issue.

However,  in testing it, I observed something -

- I used an existing row which had name field as "Oil & Gas", this had an id (auto-incremental) value of 1.
- I changed the name to "Oil / Gas", exported locally and imported on the production app.
- Import was successful but it was recorded as a new row "Oil / Gas" (id:5), instead of overwriting the existing "Oil & Gas" (id:1).

Is this how it's intended to work? 
In a case where the auto-incremental id is already used in relationships, how do I modify the name and maintain its existing id?

Thanks for your time.
